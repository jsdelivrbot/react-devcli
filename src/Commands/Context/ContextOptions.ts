export const name = 'context';

export const description = 'context description';

export const flags = {};
