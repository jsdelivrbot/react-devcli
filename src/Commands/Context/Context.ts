import {Arguments} from 'yargs';
import ACommand from '../ACommand';
import {IMakeable, IRemovable, ITestable} from '../Commands';
import {ContextConfig} from 'react-devcli';

class Context extends ACommand implements IRemovable, ITestable, IMakeable {
    public constructor() {
        const name = 'context';
        const description = 'context description';
        const options = {
        };

        super(name, description, options);
    }

    public run(argv: Arguments, config: ContextConfig): void {
        console.log(argv);
        console.log(config);
    }

    public help(argv: Arguments): void {
        console.log(argv);
    }

    public remove(argv: Arguments): void {
        console.log(argv);
    }

    public test(argv: Arguments): void {
        console.log(argv);
    }

    public make(argv: Arguments): void {
        console.log(argv);
    }
}

export default Context;
