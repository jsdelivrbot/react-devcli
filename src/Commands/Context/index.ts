import Context from './Context';

export default new Context();
