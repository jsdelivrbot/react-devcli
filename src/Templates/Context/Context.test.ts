import {js, ts} from './Context';

describe('Context template test suite', (): void => {
    test('should match context template snapshot for JS language', (): void => {
        expect(js()).toMatchSnapshot();
    });

    test('should match context template snapshot for TS language', (): void => {
        expect(ts()).toMatchSnapshot();
    });
});
