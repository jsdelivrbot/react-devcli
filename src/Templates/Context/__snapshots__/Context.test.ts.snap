// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Context template test suite should match context template snapshot for JS language 1`] = `""`;

exports[`Context template test suite should match context template snapshot for TS language 1`] = `""`;
